libs/core/state/src/collection/paginateable.ts

```diff
@@ -0,0 +1,5 @@
+export interface DaffNumericallyPaginateable {
```

Comment:
paginable?

Reply:
I was torn here. I couldn't decide on the adjective form here. I googled for 

```txt
Paginatable 
Paginateable 
Paginable 
Pageable
```

Of these four, none appear in webster's.

Reply:
I like Paginable as well.

Reply:
6a032467